erizo/src/erizo/rtp/SequenceNumberTranslator.cpp

```diff
@@ -0,0 +1,114 @@
+#include "rtp/SequenceNumberTranslator.h"
+
+#include <vector>
+
+namespace erizo {
+
+constexpr uint16_t kMaxSequenceNumberInBuffer = 511;  // = 65536 / 128 and more than 10s of audio @50fps
+constexpr uint16_t kMaxDistance = 200;
+
+DEFINE_LOGGER(SequenceNumberTranslator, "rtp.SequenceNumberTranslator");
+
+SequenceNumberTranslator::SequenceNumberTranslator()
+    : in_out_buffer_{kMaxSequenceNumberInBuffer},
+      out_in_buffer_{kMaxSequenceNumberInBuffer},
+      first_input_sequence_number_{0},
+      last_input_sequence_number_{0},
+      initial_output_sequence_number_{0},
+      initialized_{false}, reset_{false} {
+}
+
+bool SequenceNumberTranslator::lessThan(uint16_t first, uint16_t last) const {
```

Comment:
I agree, I can implement a generic RtpUtils and start adding this specific case, wdyt?